templates/base/src/__tests__/_404.test.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import Custom404 from '../pages/404';
+
+test('404 Error Page ', () => {
+  let { container, getByText, rerender } = render(<Custom404 />);
+  expect(getByText('404 - Page Not Found')).toBeInTheDocument();
```

Comment:
Good to have] getByText. --> getByHeading

Reply:
In the API, we have getByTitle not exactly something that fetches heading element directly